packages/relay/tests/lib/eth.spec.ts

```diff
@@ -313,6 +319,23 @@ describe('Eth calls using MirrorNode', async function () {
     }
   ];
 
+  const defaultLogs4 = [
+    {
+      "address": "0x0000000000000000000000000000000002131951",
+      "bloom": logBloom4,
+      "contract_id": contractId2,
+      "data": "0x",
+      "index": 0,
+      "topics": defaultNullLogTopics,
```

Comment:
nit: The log record itself can't have a `null` value for a topic of specific index.

Reply:
Done.